Dockerfile

```diff
@@ -15,9 +15,13 @@
 FROM golang:1.9
 
 ARG app_env
+ARG PRIVATE_KEY_PATH=default
```

Comment:
SUGGESTION: If you specify these as ENV rather than ARG, you can just have them passed in with the .env file

Reply:
Oh sweet, I'll change that over.

Reply:
After playing with this a bit, it appears that the Dockerfile doesn't have access to variables in the .env file (though docker-compose.yml does). Does this sound right? If so, is the idea to specify something like `ENV PRIVATE_KEY_PATH=.keys/app.rsa` in the Dockerfile?